great_expectations/experimental/column_descriptive_metrics/metrics.py

```diff
@@ -0,0 +1,39 @@
+from __future__ import annotations
+
+import uuid
+from typing import Any, Sequence, Union
+
+import pydantic
+from pydantic import BaseModel, Field
+
+
+class MetricRepositoryBaseModel(BaseModel):
+    """Base class for all MetricRepository related models."""
+
+    class Config:
+        extra = pydantic.Extra.forbid
+
+
+class Value(MetricRepositoryBaseModel):
+    value: Any  # TODO: Better than Any
+
+
+class Metric(MetricRepositoryBaseModel):
+    id: uuid.UUID = Field(description="Metric id")
+    run_id: uuid.UUID = Field(description="Run id")
+    # TODO: reimplement batch param
+    # batch: Batch = Field(description="Batch")
+    metric_name: str = Field(description="Metric name")
+    metric_domain_kwargs: dict = Field(description="Metric domain kwargs")
+    metric_value_kwargs: dict = Field(description="Metric value kwargs")
```

Comment:
What is this?
- Head: number of rows
- HeadMetric
- ZScoreUnderThreshold: double-sided, threshold
- ColumnStandardDev has a value set

Let's create Metric-specific classes for this, too